Package.resolved

```diff
@@ -1,238 +0,0 @@
-{
```

Comment:
I removed `Package.resolved`. Because it will be changed by resolving with various Xcode versions.